docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+services:
+
+  drupal:
+    container_name: drupal_admin_ui_drupal
+    environment:
+      - DRUSH_OPTIONS_URI=http://127.0.0.1
+    build:
+      context: .
+      dockerfile: ./.docker/drupal/Dockerfile
+    ports:
+      - 80:80
```

Comment:
Let's clearly document that you need to stop your local webserver and whatever is running on port 3000, as otherwise it wouldn't work. Why is everything so hard.

Reply:
:+1: